lib/kafo/data_type_parser.rb

```diff
@@ -8,9 +8,26 @@ class DataTypeParser
 
     def initialize(manifest)
       @logger = KafoConfigure.logger
-      @types = {}
+
+      lines = []
+      last_line = ''
       manifest.each_line do |line|
-        if (type = TYPE_DEFINITION.match(line.force_encoding("UTF-8")))
+        line = line.force_encoding("UTF-8").strip
+        next if line.start_with?('#')
+
+        line = line.split(' #').first.strip
+        if line =~ TYPE_DEFINITION
+          lines << last_line
```

Comment:
What's the idea behind the `last_line` here? When is that ever relevant for a type definition?

Reply:
Well, it's a buffer where I store all the lines in one string to drop the newlines.

I renamed the variable now, to make it more obvious